libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartSyncPlugin/SFSmartSyncPlugin.m

```diff
@@ -64,9 +64,8 @@ - (SFSmartSyncSyncManager *) globalSyncManager
 - (void) resetSyncManager
 {
     self.syncManager = nil;
-    SFUserAccount* user = [SFUserAccountManager sharedInstance].currentUser;
-    [SFSmartSyncSyncManager removeSharedInstance:user];
-    self.syncManager = [SFSmartSyncSyncManager sharedInstance:user];
+    self.globalSyncManager = nil;
+    [SFSmartSyncSyncManager removeSharedInstances];
```

Comment:
We were not cleaning up the global sync manager - for that reason the sync test with global store would fail (when more than one of them was run)